common.lic

```diff
@@ -155,6 +159,12 @@ module DRC
         put message
         timer = Time.now
         next
+      when /^You must be standing/, /^You should stand up first/, /You'll need to stand up first/, /^You can't do that while (sitting|kneeling|lying)/, /^You should be sitting up/, /^You really should be standing to play/
```

Comment:
Miss `/You'll need to stand up first/` => `/^You'll need to stand up first/`.

Reply:
Blech, I need to pay better attention.